src/context/favoriteContext.tsx

```diff
@@ -1,6 +1,5 @@
-import React, { useEffect } from "react";
+import React from "react";
 import { IFavoriteContext } from "../types/contextInterface";
```

Comment:
You are declaring that this is already type definition folder in the folder name.
```suggestion
import { IFavoriteContext } from "../types/context";
```